ui/src/CollectionExerciseDetails.js

```diff
@@ -181,16 +181,14 @@ class CollectionExerciseDetails extends Component {
     }
 
     const newActionRule = {
-      id: uuidv4(),
       type: this.state.newActionRuleType,
       triggerDateTime: new Date(
         this.state.newActionRuleTriggerDate
       ).toISOString(),
-      hasTriggered: false,
       classifiers: this.state.newActionRuleClassifiers,
+      packCode: this.state.newActionRulePackCode,
       printTemplate: printTemplate,
```

Comment:
This printTemplate isn't being used by the in the java code so does it need to be here?

Reply:
yeah, you're right... my bad